conversation_reconstruction/dataflow/write_page_states_to_spanner/spanner_write_test.py

```diff
@@ -4,21 +4,25 @@
 from __future__ import division
 from __future__ import print_function
 import unittest
-from write_utils.write import SpannerWriter 
+import os
+from write_utils.write import SpannerWriter
 from google.gax import retry
 
 
 class SpannerWriteTest(unittest.TestCase):
 
   def test_spanner_write(self):
     writer = SpannerWriter('wikiconv', 'page_info')
-    writer.create_table('page_states', ('page_id', 'authors',
-                                       'conversation_id', 'deleted_comments',
-                                       'page_state', 'rev_id', 'timestamp'))
+    writer.create_table('page_states', ({'page_id': "STRING", 'authors':"STRING",
+                                         'conversation_id': "STRING", 'deleted_comments': "ARRAY(STRING)",
```

Comment:
I don't think I understand what, of your code's logic, is being tested here...? Looks like your testing generic spanner logic?

Reply:
I have a write_utils package, this is mainly testing the package.